cmd/down.go

```diff
@@ -99,22 +113,41 @@ func Down() *cobra.Command {
 }
 
 func allDown(ctx context.Context, manifest *model.Manifest, rm bool) error {
+	spinner := utils.NewSpinner("Deactivating your development containers...")
+	spinner.Start()
+	defer spinner.Stop()
+
 	if len(manifest.Dev) == 0 {
 		return fmt.Errorf("okteto manifest has no 'dev' section. Configure it with 'okteto init'")
 	}
+
+	c, _, err := okteto.GetK8sClient()
+	if err != nil {
+		return err
+	}
+
 	for _, dev := range manifest.Dev {
-		if err := runDown(ctx, dev, rm); err != nil {
-			analytics.TrackDown(false)
-			err = fmt.Errorf("%w\n    Find additional logs at: %s/okteto.log", err, config.GetAppHome(dev.Namespace, dev.Name))
+		app, _, err := utils.GetApp(ctx, dev, c, false)
+		if err != nil {
 			return err
 		}
+
+		if apps.IsDevModeOn(app) {
+			spinner.Stop()
```

Comment:
there is a `defer spinner.Stop()`, you don't need it here, right?

Reply:
if I don't stop the spinner at this point, it will repeatedly swap the spinner message of the allDown function with that of the runDown function in a rather 'crazy' way.

Reply:
maybe you should pass the spinner as an argument to `runDown`?

Reply:
yes, it could be an option. Another option I have thought of is to create a struct with all these options available in the command: (like upOptions in the `up` command) to avoid functions with too many parameters. Maybe it would be cleaner.